.travis.yml

```diff
@@ -45,7 +47,7 @@ addons:
       - flex
       - g++
       - git
-      - libboost1.55-all-dev
+      - libboost1.58-all-dev
```

Comment:
`libboost1.58-all-dev` is not found in Ubuntu Xenial so I had to change it to use a more recent version.